tests/test_learning.py

```diff
@@ -52,23 +52,33 @@ def test_decision_tree_learner():
 
 def test_neural_network_learner():
     iris = DataSet(name="iris")
-    iris.remove_examples("virginica")
-    
+
     classes = ["setosa","versicolor","virginica"]
-    iris.classes_to_numbers()
+    iris.classes_to_numbers(classes)
+
+    nNL = NeuralNetLearner(iris, [5], 0.15, 75)
+    pred1 = nNL([5,3,1,0.1])
+    pred2 = nNL([6,3,3,1.5])
+    pred3 = nNL([7.5,4,6,2])
 
-    nNL = NeuralNetLearner(iris)
-    # NeuralNetLearner might be wrong. Just check if prediction is in range.
-    assert nNL([5,3,1,0.1]) in range(len(classes))
+    # NeuralNetLearner might be wrong. If it is, check if prediction is in range.
+    assert pred1 == 0 or pred1 in range(len(classes))
```

Comment:
This is ok for now. But in the long run, two things:
(1) There should be a way to test any learner, not just NeuralNetLearner
(2) To deal with the possibility of nondeterministic learners getting it wrong, there shoud be experiments, with overall results graded (for example, mst get at least 2 out of 3 answers right).